apcd-cms/src/apps/admin_submissions/views.py

```diff
@@ -38,8 +38,8 @@ def get_context_data(self, *args, **kwargs):
             **s,
             'status': title_case(s['status']),
             'outcome': title_case(s['outcome']),
-            'received_timestamp': datetime.strptime(s['received_timestamp'], '%Y-%m-%dT%H:%M:%S.%f'),
-            'updated_at': datetime.strptime(s['updated_at'], '%Y-%m-%dT%H:%M:%S.%f'),
+            'received_timestamp': parser.parse(s['received_timestamp']),
```

Comment:
Has this parse statement been tested with our DB? I noticed that you're not setting a parse object.